skfp/applicability_domain/standard_deviation.py

```diff
@@ -0,0 +1,137 @@
+from numbers import Real
+
+import numpy as np
+from sklearn.utils._param_validation import Interval
+from sklearn.utils.validation import validate_data
+
+from skfp.bases.base_ad_checker import BaseADChecker
+
+
+class StdEnsembleADChecker(BaseADChecker):
+    """
+    Standard deviation ensemble method.
+
+    Defines applicability domain based on the spread (standard deviation) of predictions
+    from individual estimators in an ensemble model [1]_. The method assumes that
+    if different estimators give similar predictions for a given molecule,
+    the prediction is more likely to be reliable (i.e., in-domain).
+
+    This approach requires an ensemble model exposing the ``estimators_``
+    attribute (e.g., RandomForestRegressor), where each
+    sub-model must implement a ``predict(X)`` method. At prediction time,
+    each sample is passed to all estimators, and the standard deviation of
+    their predictions is calculated. The sample is considered in-domain if
+    the standard deviation is lower than or equal to the specified threshold.
+
+    Typically, physicochemical properties (continous features) are used as inputs.
+    Consider scaling, normalizing, or transforming them before computing AD to lessen
+    effects of outliers, e.g. with ``PowerTransformer`` or ``RobustScaler``.
+
+    This method is particularly useful in assessing uncertainty in ensemble
+    learning-based QSAR models.
```

Comment:
Merge this with the first paragraph, since here you basically say `This method assesses model uncertainty, linked to the model reliability, which is inherently higher in densely sampled regions inside training data.`